PORTAL/jobs/_current.py

```diff
@@ -75,7 +75,11 @@ def __init__(self, reqid, msg):
 class RequestNotPendingError(StagingRequestError):
 
     def __init__(self, reqid, status=None):
-        super().__init__(reqid, f'could not stage {reqid} (expected pending, got {status or "???"} status)')
+        super().__init__(
+            reqid,
+            f'could not stage {reqid} '
+            f'(expected pending, got {status or "???"} status)'
+        )
```

Comment:
Implicit string literal concatenation always scares me.  If we do have to split it, could we at least make it more obvious?

```suggestion
        super().__init__(
            reqid,
            (f'could not stage {reqid} '
             f'(expected pending, got {status or "???"} status)'),
        )
```

or better:

```suggestion
        msg = f'could not stage {reqid} (expected pending, got {status or "???"} status)'
        super().__init__(reqid, msg)
```

or 

```suggestion
        _status = status or '???'
        msg = f'could not stage {reqid} (expected pending, got {_status} status)'
        super().__init__(reqid, msg)
```